client/src/components/Display.spec.js

```diff
@@ -0,0 +1,22 @@
+import "jest-dom/extend-expect";
+import React from "react";
+import { cleanup, render } from "react-testing-library";
+import Display from "./Display";
+
+afterEach(cleanup);
+
+describe("<Display/>", () => {
+  it("renders the display component", async () => {
+    const { getByTestId, debug } = await render(<Display currentState />);
```

Comment:
Great job figuring this out, this went on to help at least like 6 people in our group.